dataland-frontend/src/components/resources/dataTable/conversion/Utils.ts

```diff
@@ -37,3 +37,16 @@ export function getFieldValueFromDataModel(identifier: string, dataModel: any):
   }
   return currentObject;
 }
+
+/**
+ * Maps the technical name of a select option to the respective original name
+ * @param technicalName of a select option
+ * @param mappingObject that contains the mappings
+ * @returns original name that matches the technical name
+ */
+export function getOriginalNameFromTechnicalName<T extends string>(
```

Comment:
Why is thies necessary? Is this for the translation Yes <-> Ja?

Reply:
No it's not.

Check for its usages in the view config and you will understand why it is needed.  It is pretty simple.

Reply:
ok, will check